### ISSUE TITLE
[Design] Hall of Fame Options

### ISSUE THREAD

**Issue by tonyxj** [psybuzz/goldenconversations#40]
Hi Guys,
Here are six design directions for the Hall of Fame page. Let me know which one(s) you prefer.
I'll be doing another set of interactions once I've gathered your feedback. @psybuzz @nikilselvam 
![image](https://cloud.githubusercontent.com/assets/6765882/5658680/e59125fe-96ba-11e4-8dff-78d1d974adc8.png)

The top three are one set of related directions. The bottom three are another set. 
The two sets could work together. For example clicking on a number in the second set takes you to the first set kind of UI, where actual conversations are laid out. The bottom is almost like a stats view. It's an area that is meant to give some insights, for the curious type. 

Since the idea of **golden conversations** is such a key idea to our app, I think it makes sense to surface some copy / writing as to why some conversations have golden labels and are kept sticky near the top.

**Comment by nikilselvam** [comment]
I like the the top row of designs better than the second row. I agree that the numbers in the second row add some intrigue from a stats perspective, but I also think that at first glance it could be confusing. It doesn't feel very intuitive in my opinion to grasp the value of the numbers associated with each archived conversation when a user may not have experience with our website. By contrast, the first and third designs in the top row look fantastic and are more intuitive. I think that users will quickly recognize and understand the color gradient in the first design. Meanwhile, the third design leverages the convershapes that the user will already be familiar with and provides a nice visual view.

I think that a combination of the first and third designs would be great. We could merge the strong color gradient of the first design and add in the convershapes denoting how many people in the conversation have archived the conversation of the third design. The tricky part may be choosing colors for the vertices in the convershapes without using shades of gold (since the conversation itself will be in a shade of gold), but if we can find some nice colors here, I think that this combined layout will look very nice.

What do you guys think? @tonyxj @psybuzz

**Comment by psybuzz** [comment]
In the top row, I also think the first and third designs seem like interesting directions to explore.  The left design's colorful style really stood out, and I like the 3rd design's use of shapes.  At first glance, I didn't notice the third design's shapes at all, cause the colors were so close to white.  It was only after I clicked on the Github image saw it full size that I noticed it.

I think having a visually noticeable first glance can be a good goal, because users would immediately know whether they're on the Hall Of Fame page or on the Regular page.  However, if someone's hall of fame only has convo's with 1 person who 'favorited' it, then would all the convo's have the same background?

For the second row, I think it might be pretty neat to see stats.  However, to me, they seem pretty huge relative to everything else, but I think over more variation and iteration, they could be even more appealing!

Also, what do you guys think about using radial circles to convey information about how many people 'archived' a convo?
![](http://cdn.pttrns.com/pttrns/2158/original/Eidetic-1.png)

**Comment by tonyxj** [comment]
@psybuzz I think the donut is a very interesting idea. One thing I like in particular is the donut for showing percentage breakdown. For example, the online tool treehouse, has a particularly nice visual for displaying points you've earned for learning stuff.
![image](https://cloud.githubusercontent.com/assets/6765882/5698073/1fb6490c-99b9-11e4-8355-7b93e6df74c4.png)

**Comment by tonyxj** [comment]
Great feedback guys! @nikilselvam @psybuzz 
I've taken what you've said and incorporated into another iteration of designs. This time there is a side panel with some copy explain the idea behind GC as well as a mini breakdown of how the user's been using the app.
The convershapes™ are also incorporated into the design, using a combination of inner color and border color to signify whether one has saved it to HOF. Be sure to click on the image for an enlarged view. 
![image](https://cloud.githubusercontent.com/assets/6765882/5698096/aeede5c6-99b9-11e4-913f-24c77bf8731d.png)

**Comment by nikilselvam** [comment]
I like this design! The convershapes + color scheme looks great. Also lol for trademarking convershapes in your message above @tonyxj 

I like the three tiers: HOF conversations, gold plated conversations, and golden conversations. One suggestion I have is to flip the order of filled vs. unfilled vertices in the convershapes. It looks like a filled vertex means that someone hasn't archived a conversation yet while an unfilled vertex means that someone has archived a conversation.

I think that this may be counterintuitive: having an unfilled vertex represent someone who hasn't archived a conversation and a filled vertex represent someone who has archived a conversation feels more natural to me. I like the idea of archiving a conversation representing the act of filling in the vertex.

What do you guys think?

**Comment by psybuzz** [comment]
Neat work!  The feeling i get seeing the conversations is a bit like seeing highlighted stories in a news journal.  Things of importance.  I like it.

@nikilselvam brings up a good point about the convershapes use of filled vs unfilled circles.  I guess my intuition was that the unfilled circles were people who archive/favorited the convo, because the background was yellow.  I do think it could be established more clearly.  What if we moved the convershapes outside of the box, and moved them over to the left?  That way, they'd all be on top of a white background?  Do you guys think that would that change the distinction?

Small nit: on the right side, it says "What is golden conversations?"  Perhaps it might be more natural to say "What are golden conversations?"

I'm also really liking the stats circle on the side.  By seeing a circle thats partially gold, I really want to see it filled gold all the way round.  Sort of like some aesthetic incentive to make more golden conversations, I guess, but maybe that's just me!  Also, gold/silver circles remind me of the olympics, which is related to Halls of Fame!

Another question for @tonyxj :  I see that there is a small gap between each convo entry, giving some whitespace between convos.  How does it look when there is no gap?  I imagine that, if someone has a page full of Gold convo's, scrolling through them seeing a gap between each one, vs seeing a continuous block of yellow would be very different.  Is there a reason why you chose to make a gap?  Or is it simply an artistic decision that comes with experience and creative judgment?

**Comment by tonyxj** [comment]
Great point guys! 
I think the idea is that each node on a convershape would be initally black, but if one flips on the HOF switch, that node would become golden. The difficulty here is that gold is such a bright color, having it on a white background is hard to see. In this sense the border, is there to highlight the gold from the white background. I think moving it out of the background gold might improve its effect. I've made the change below. One thing about keep it in the background was that putting it outside of the gold padding might compromise the spatial association between the shapes and the title, as the convershapes might appear to be distant from the text. However, removing the padding gutters between those two would make things feel cramped. 

Great question about the gaps between each listing, no gap is a different approach. Each has its merits. With no gaps, the colors indicate areas where the conversations belong to, it's like a habitat. The approach with gaps is modelled after buttons and labels -- it's akin to applying a sticker label to each conversation. Each conversations would maintain it's own label, grouped together, instead of being kept together in in the gold colored habitat, as exemplified by the no gap approach. 
![image](https://cloud.githubusercontent.com/assets/6765882/5699403/9441153c-99db-11e4-96a1-7ac476ec4149.png)
![image](https://cloud.githubusercontent.com/assets/6765882/5699404/9d358f6a-99db-11e4-980e-2bc82617d6cc.png)

**Comment by tonyxj** [comment]
@nikilselvam @psybuzz 
Guys, out of the above, which one do you prefer?

**Comment by psybuzz** [comment]
Well Tony, I think they're both good designs!  I've been staring at the two, and I'm starting to like the second one more, because the slight gap really does make it easy to get a sense of how many convos are in each category.  I'd reaaaaly be fine with either, but if I had to choose, I'd go with the 2nd.

That said, if the only difference between the two designs is the small gap, it's something that's pretty easy to change later on if needed.  What's your opinion @nikilselvam ?

**Comment by nikilselvam** [comment]
Hm good question. I like both of them as well! I liked the first one more initially, but similar to Erik, I'm starting to like the second one more as I continue looking at it. I'd be fine either way! What do you think @tonyxj ?

One other thought: having the convershapes next to each conversation looks really good. It adds more personality and clarity to each conversation, and to some extent it excites me to want to explore each conversation further. I think we should consider adding these to the conversations on the home page as well. What do you guys think?

**Comment by tonyxj** [comment]
Great questions guys. Apologies for the delayed response, really been hammered hard by assignments and work lately. I like the second option better too, having a subtle yet distinct margin allows people's minds to perceive (subconsciously) that each block is a separate conversation. 

@nikilselvam you brought up a good point. Having that convershapes does bring out the personality to each convo. Since we already it in the messaging view, it'd be good to port it over to the homepage and HOF. Let's make it a goal, maybe a stretch goal, since the tasks are adding up in Trello.

Let's move forward with this design for the HOF. I'll be opening up a new branch for this using placeholder HTML for now.

**Comment by tonyxj** [comment]
In PR. Closing ticket.
